miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditRunController.java

```diff
@@ -312,4 +320,85 @@ public String processSubmit(@ModelAttribute("run") Run run, ModelMap model, Sess
       throw ex;
     }
   }
+
+  @Value("${miso.run.report.links:}")
+  private String runReportLinksConfigLine;
```

Comment:
Rather than wiring in the raw values here and parsing them for every request, could we have a "UriBuilder" type of class with a `getUri(Run)` method, and initialise these during startup? Startup can then fail if there's a bad config. This could be done in `MisoAppListener`, or preferably `WebConfig`